config/config-defaults.yaml

```diff
@@ -87,6 +87,10 @@ data:
     # no default-resolver-type is specified by default
     default-resolver-type:
 
+    # default-imagepullbackoff-timeout contains the default number of minutes to wait
+    # before requeuing the TaskRun to retry
+    # default-imagepullbackoff-timeout: "5"
```

Comment:
We probably want to use time values instead, like `1m`, `5m`, `10s` or `1h`.

Reply:
or we need to add "minutes" in the field name.

Reply:
I was looking at this too and thinking that in all cases I can think of K8s uses "seconds". I think that's simplest vs. supporting something fancier.

Reply:
good point -- we might want to keep our time units consistent across the board in the future and I think `time.Duration` is truly better basis.